pkg/runctl/cfg/config.go

```diff
@@ -84,11 +84,16 @@ type PipelineRunsConfigStruct struct {
 	// NetworkPolicies maps network profile names to network policies.
 	// Each value is a Kubernetes network policy manifest in YAML format.
 	NetworkPolicies map[string]string
+
+	// TaskNamespace is the name of the namespace containing the task definition.
+	TaskNamespace string
```

Comment:
Shouldn't we also add the task name here, for consistency?

And I would then really take load the values from the config map instead of hardcoding it.

Reply:
Done